config/application.yml.default

```diff
@@ -111,6 +111,7 @@ encrypted_document_storage_s3_bucket: 'test-bucket-changeme'
 event_disavowal_expiration_hours: 240
 feature_idv_force_gpo_verification_enabled: false
 feature_idv_hybrid_flow_enabled: true
+frontend_analytics_enabled: true
```

Comment:
1) Is there a way to make the analytics disabled config only apply in the test environment to avoid affecting prod,
or
2) Is this something that could happen in prod and we want to protect against it there too